bosh-release/packages/mysqldriver/packaging

```diff
@@ -1,13 +0,0 @@
-# abort script on any command that exit with a non zero value
```

Comment:
What would be the reason for removing this file?

Reply:
@imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

Reply:
The PR shows some files have been moved but some as deleted and added as new.